backend/de.metas.payment.esr/src/main/java/de/metas/payment/esr/api/impl/ESRImportDAO.java

```diff
@@ -315,6 +307,16 @@ private I_C_ReferenceNo fetchESRInvoiceReferenceNumber(@NonNull final String esr
 				.iterate(I_ESR_Import.class);
 	}
 
+	@Override
+	public Iterator<I_ESR_ImportFile> retrieveActiveESRImportFiles(final Properties ctx, final int orgId)
+	{
+		return queryBL.createQueryBuilder(I_ESR_ImportFile.class, ctx, ITrx.TRXNAME_None)
```

Comment:
wdyt about kicking out ctx as parameter and do
```suggestion
	public Iterator<I_ESR_ImportFile> retrieveActiveESRImportFiles(@NonNull final OrgId orgId)
	{
		return queryBL.createQueryBuilderOutOfTrx(I_ESR_ImportFile.class)
```

?